jest.config.cjs

```diff
@@ -0,0 +1,214 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
A *.cjs file, haven't seen one of those in a while!
If we can't convert this to .ts or .js we should make sure this is getting picked up by ESLint+Prettier workflows.

Reply:
Good catch, f1e3cbf92.

Unfortunately Jest can't run in an MJS context, which is the default for `.js` and `.ts` files in our project (because of the `"type": "module"` in our `package.json`), so using a different extension results in errors like this one:

```
SyntaxError: Cannot use import statement outside a module
```